packages/composite-checkout-wpcom/src/lib/translate-cart.ts

```diff
@@ -52,10 +55,22 @@ export function translateWpcomCartToCheckoutCart( serverCart: ResponseCart ): WP
 		},
 	};
 
+	const subtotalItem: CheckoutCartItem = {
+		id: 'total',
```

Comment:
I'm a little confused that `totalItem` and `subtotalItem` have the same id, which probably stems from the inconsistency on the backend about exactly what the subtotal is. I'm not sure what would be a better alternative though. IIUC this is "total cost before partial credits and before taxes".

Reply:
Oops. That was a typo. Thanks!